documentation/adr/0005_using_buildpacks_on_paas.md

```diff
@@ -0,0 +1,40 @@
+# Using Buildpacks on PaaS Instead of Containers
+
+**Date: 2020-04-14**
+
+## Status
+
+**Decided**
+
+## Parties Involved
+
+ * Brendan Quinn
+ * Colin Saliceti
+ * Davide Dippolito
+ * Todd Tyree
+
+## Context
+
+We chose GOV.UK PaaS to simplify our infrastructure and reduce our time to deploy and our ongoing maintenance overheads.
+In order to make the most efficient use of GOV.UK PaaS, we will use their officially supported  
+[buildpacks](https://docs.cloud.service.gov.uk/deploying_apps.html#deploy-an-app-to-production). This the only way to
+deploy to GOV.UK PaaS in a fully-supported way. 
+
+## Decision
+
+TVS will be run using buildpacks when it goes live on GOV.UK PaaS.
```

Comment:
I would add this is a temporary agreement to ease the transition and this will be revisited to support Docker as it's the strategic direction of DfE.

Reply:
👍